### PR TITLE
Adds Google Tag Manager

### PR DESCRIPTION
**Pull request by jonnilundy**
Adding Google Tag Manager by the suggestion from Mark Hensley so that we can implement Hotjar.

### CODE REVIEW

File: node/electric/src/layouts/base.soy [hunk 1]
```diff
@@ -39,8 +42,11 @@
 
 	</head>
 	<body data-senna>
-		<div data-senna-surface id="wrapper">
-			<div class="senna-loading-bar"></div>
+
+  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KLQSNVD"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```
Comment (jonathantneal) [inline node/electric/src/layouts/base.soy]: Nice! Would you please add a space between the `src` and `height` attributes?

File: node/electric/src/layouts/base.soy [hunk 2]
```diff
@@ -39,8 +42,11 @@
...
+
+    <div data-senna-surface id="wrapper">
+      <div class="senna-loading-bar"></div>
```
Comment (jonathantneal) [inline node/electric/src/layouts/base.soy]: Is this part of the Google Tag Manager?

Comment (jonnilundy) [reply node/electric/src/layouts/base.soy]: As far as I understand, this is an iframe that you can initiate from their console to preview your triggers before deploying changes.

**Comment by jonnilundy** [comment]
@zenorocha @vitorfernandes Once this is added, Mark Hensley said he could help us set up heat mapping.

File: node/electric/src/layouts/base.soy [hunk 3]
```diff
@@ -39,8 +42,11 @@
 
 	</head>
 	<body data-senna>
-		<div data-senna-surface id="wrapper">
-			<div class="senna-loading-bar"></div>
+
+  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KLQSNVD" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```
Comment (zenorocha) [inline node/electric/src/layouts/base.soy]: Indentation looks a little bit off on this line

**Comment by jonnilundy** [comment]
@zenorocha I updated the indentation in the last commit. Is there any other feedback you have before merging?

**Pull request merged**